docs/markdown/Python/python-goals/python-check-goal.md

```diff
@@ -186,6 +186,30 @@ MY_SETTING = URLPattern(pattern="foo", callback=lambda: None)
 python_source(name="django_settings", source="django_settings.py")
 ```
 
+> 🚧 Importing from `extra_type_stubs`
```

Comment:
Should this be removed now that we have the `help` text? The downside of this is that more docs can be a bad thing - it makes it harder to find stuff and makes the project look more overwhelming.

Reply:
I would vote to definitely keep this section, because:
1. The docs are already talking about `extra_type_stubs`, and without this section I feel the info presents an incomplete picture
2. There is more info about dealing with the limitation here than there is in the `help` text

I'm less opinionated about keeping the NOTE in the help text 😅